physionet-django/project/modelcomponents/submission.py

```diff
@@ -1,7 +1,9 @@
+from django.conf import settings
```

Comment:
are those new imports needed?

Reply:
thanks for pointing this out, not needed